client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -294,19 +296,6 @@ class PluginMeta extends Component {
 		}
 	}
 
-	getDefaultActionLinks( plugin ) {
-		let adminUrl = get( this.props, 'selectedSite.options.admin_url' );
-		const pluginSlug = get( plugin, 'slug' );
-
-		if ( pluginSlug === 'vaultpress' ) {
-			adminUrl += '/admin.php?page=vaultpress';
-		}
-
-		return adminUrl
-			? { [ i18n.translate( 'WP Admin' ) ]: adminUrl }
-			: null;
-	}
-
```

Comment:
Ah, so this is where that method is coming from! 😄 Should've scrolled down :slightly_smiling_face: